scripts/check-public-api.ts

```diff
@@ -251,7 +251,7 @@ export async function parseIndexFile(filePath: string): Promise<string[]> {
     /export \* from '([\w/.]+)'/g,
     fileContent
   );
-  const starFileExports = await Promise.all(
+  const starFileExports =  await Promise.all(
```

Comment:
```suggestion
  const starFileExports = await Promise.all(
```

Reply:
[info] The linting is a bit off (also in the .spec file). I don't exactly remember why we exclude scripts/ in .eslintrc